client/src/featureform/register.py

```diff
@@ -1858,7 +1848,7 @@ def register_spark(self,
         self.__resources.append(provider)
         return OfflineSparkProvider(self, provider)
 
-    def register_k8s(self,
+    def register_kubernetes(self,
```

Comment:
Design approved by simba had this as k8s. Would leave as is until we have further discussion